driver/motor.cpp

```diff
@@ -5,8 +5,20 @@ extern "C"
 #include "wiringPi/wiringPi.h"
 }
 
-int measure_motor()
+// 0 ~ 100のパーセンテージを引数として
+// モーターの出力を制御する。
+// min 0%のとき0、max 100%のとき100を引数とする。
+int motor_control(unsigned int powerPercentage)
 {
+  // ガード処理
+  if(powerPercentage < 0){
+    powerPercentage = 0;
+  }else if(powerPercentage > 100){
+    powerPercentage = 100;
+  }else{
+    //do nothing
+  }

```

Comment:
It is better to use `std::clamp` instead of L14 ~ L20 like this:
`const auto powerPercentage_ = std::clamp(powerPercentage, 0, 100);`

You **have to** add `#include <algorithm>`.